.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: python
+python:
+  - "2.7"
+  - "3.4"
```

Comment:
The `3.4` has officially retired https://www.python.org/downloads/release/python-3410/
```suggestion
```